spec/features/job_seekers_can_use_current_location_link_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.feature 'Using the current location shortcut link', js: true do
+  let(:dfe_postcode) { 'SW1P 2LU' }
+  let(:dfe_lat) { 51.4980026 }
+  let(:dfe_lon) { -0.1323527 }
+
+  before do
+    visit root_path
+    simulate_location
+  end
+
+  def mock_postcodes_io(response_type = 200, null = false)
+    fixture_path = "postcodes_io_#{response_type}#{'_null' if null}.json"
+    response_text = response_type == 200 ? file_fixture(fixture_path).read : '""'
+
+    page.execute_script <<-JS
+      #{file_fixture('jquery.mockjax.min.js').read}
+      $.mockjaxSettings.logging = 0;
+
+      $.mockjax({
+        url: "https://api.postcodes.io/postcodes",
+        status: #{response_type},
+        responseText: #{response_text}
+      });
+    JS
```

Comment:
Could we have mocked the postcodes.io calls using the VCR gem? (https://github.com/vcr/vcr) 

Mainly asking out of curiousity